types/landisgyr/T550/meta.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "ULTRAHEAT T550",
+  "name": "WZU-LR",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed